### PR TITLE
feat: Flag to set absolute path for explainerfile implemented

### LINKED ISSUE #221: explainerfile in dashboard.yaml must be absolute or relate to hub working directory
Hi!

I'm trying to reproduce example from documentation https://explainerdashboard.readthedocs.io/en/latest/hub.html#adding-dashboards-using-url and find next issue:

With next code
```code
db1 = ExplainerDashboard(explainer, title="First Dashboard", name="db1")
db1.to_yaml("dashboards/dashboard1/dashboard.yaml", dump_explainer=True)

db2 = ExplainerDashboard(explainer, title="Second dashboard", name="db2")
db2.to_yaml("dashboards/dashboard2/dashboard.yaml", dump_explainer=True)

hub = ExplainerHub([db1], max_dashboards=5,
                   add_dashboard_route=True,
                   add_dashboard_pattern="dashboards/{}/dashboard.yaml")
hub.run()
```

in files dashboards/dashboard1/dashboard.yaml and dashboards/dashboard2/dashboard.yaml key "explainerfile" saves with value "explainer.joblib" and if i'm trying to add this dashboard to Hub (request http://localhost:8050/add_dashboard/dashboard2) I see no errors in log, request "GET /add_dashboard/dashboard2 HTTP/1.1" 302 -, but no dashboards created in UI. But if I manually change "explainerfile: explainer.joblib" to "explainerfile: dashboards/dashboard2/explainer.joblib" in "dashboards/dashboard2/dashboard.yaml" than it works fine and i see "Second dashboard" in UI.

I think that in "to_yaml" method need change logic to add "dashboard_path" to "explainerfile" or make it optional with flag "explainerfile_absolute_path=True".

### PR DESCRIPTION
**Pull request by jenoOvchi**
Fixes #221

### CODE REVIEW

File: explainerdashboard/dashboards.py [hunk 1]
```diff
@@ -749,6 +751,9 @@ def to_yaml(self, filepath=None, return_dict=False,
         if filepath is not None:
             dashboard_path = Path(filepath).absolute().parent
             dashboard_path.mkdir(parents=True, exist_ok=True)
+            
+            if explainerfile_absolute_path:
+                dashboard_config["dashboard"]["explainerfile"] = str(dashboard_path) + '/' + str(explainerfile)
```
Comment (KawinL) [inline explainerdashboard/dashboards.py:756]: Should we use os.path.join instead?

Comment (jenoOvchi) [reply explainerdashboard/dashboards.py:756]: > Should we use os.path.join instead?

Sure, why not.

Comment (jenoOvchi) [reply explainerdashboard/dashboards.py]: Done

**Comment by oegedijk** [comment]
replaced os.path.join with the equivalent pathlib functionality and also low user to specify a specific absolute path

**Comment by oegedijk** [comment]
Thanks a lot for the contribution! Hope it will prove useful!

**Comment by jenoOvchi** [comment]
Thank you!

**Comment by oegedijk** [comment]
just released it as version 0.4.0

**Pull request merged**